index.md

```diff
@@ -273,12 +253,14 @@ Edit the text to match who can attend the workshop. For instance:
 - This workshop is open to the public.
 - If you are interested in attending this workshop, contact me@example.com
   for more information
+{% endcomment %}
 
 <p id="who-can-attend">
     <strong>Who can attend?:</strong>
-    This workshop is open to ....
+    The course is aimed at graduate students and other researchers. It is assumed that participants already write code for
+    their research, but no expertise is required. Familiarity with The Bash Shell is expected.
```

Comment:
A basic knowledge of The Unix Shell [https://swcarpentry.github.io/shell-novice/](https://swcarpentry.github.io/shell-novice/) is expected

Reply:
it is revised.